docs/src/components/Layout/LogoLink.tsx

```diff
@@ -0,0 +1,21 @@
+import Link from 'next/link';
+
+import { VisuallyHidden } from '@aws-amplify/ui-react';
+
+import { Logo } from '../Logo';
+
+interface LogoLinkProps {
+  platform: string;
+  onClick?: () => void;
+}
+
+export const LogoLink = ({ platform, onClick }: LogoLinkProps) => {
+  return (
+    <Link href={`/${platform}`} passHref>
+      <a onClick={onClick} className="docs-logo-link">
```

Comment:
I noticed this link has a default focus ring color that doesn't match our theme

<img width="148" alt="Screen Shot 2022-05-25 at 6 00 51 PM" src="https://user-images.githubusercontent.com/376920/170374721-27b234b8-1230-496e-869e-1f85ae99a92d.png">

Reply:
Should be fixed now, added focus styling to all `a` tags